substrate/frame/revive/rpc/dockerfiles/eth-rpc/Dockerfile

```diff
@@ -1,23 +1,17 @@
-FROM rust AS builder
-
-RUN apt-get update && \
-		DEBIAN_FRONTEND=noninteractive apt-get install -y \
-		protobuf-compiler \
-		clang libclang-dev
+FROM docker.io/paritytech/ci-unified:latest AS builder
 
 WORKDIR /polkadot
 COPY . /polkadot
-RUN rustup component add rust-src
```

Comment:
This is needed to build the fixtures. But they are only needed when `runtime-benchmarks` are enabled for the runtime. So I guess its fine to remove it.

Reply:
ah right forgot to remove it